server/autolinkclient/client.go

```diff
@@ -64,3 +82,47 @@ func (c *Client) Add(links ...autolink.Autolink) error {
 
 	return nil
 }
+
+func (c *Client) Get(autolinkName string) ([]*autolink.Autolink, error) {
+	queryParams := url.Values{
+		AutolinkNameQueryParam: {autolinkName},
+	}
+
+	resp, err := c.call("/"+autolinkPluginID+"/api/v1/link", http.MethodGet, nil, queryParams)
+	if err != nil {
+		return nil, err
+	}
+
+	respBody, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("unable to install autolink. Error: %v, %v", resp.StatusCode, string(respBody))
+	}
+
+	var response []*autolink.Autolink
+	if err = json.Unmarshal(respBody, &response); err != nil {
+		return nil, err
+	}
+
+	return response, nil
+}
+
+func (c *Client) call(url, method string, body []byte, queryParams url.Values) (*http.Response, error) {
+	req, err := http.NewRequest(method, url, bytes.NewReader(body))
+	if err != nil {
+		return nil, err
+	}
+
+	req.URL.RawQuery = queryParams.Encode()
+
+	resp, err := c.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
```

Comment:
This would forbid the caller from reading the response body, so it should be the caller's responsibility to call the close method